lib/routes.js

```diff
@@ -1,6 +1,7 @@
 var loggedInAction = function loggedInAction() {
   if (!Meteor.userId()) {
-    this.redirect('login');
+    this.redirect('/signup');
+    this.next();
```

Comment:
This isn't needed if the `else block does it as well. Can remove from both places and have `this.next();` outside of the `if`/`else`.